### DISCUSSION TITLE
need more options?

### DISCUSSION THREAD

**Discussion by ninmba** [WerWolv/ImHex#944] [Get Help]
Hello, how about adding more options like:
1- let imHex to start with maximize window by default?
2- add checkbox option to not store opened recently files?

**Comment by iTrooz** [comment]
1- This seems like something that should be handled at the environment level, not ImHex. Maybe make a script to launch ImHex, get its window, and resize it
2 - Seems like a good idea indeed. @WerWolv do you agree ?

**Reply by WerWolv** [reply]
We already have an option for the second request. It's Under `Extras -> Settings -> General -> Save recently used providers`. Maybe it needs a better name?

**Reply by Calinou** [reply]
> 1- This seems like something that should be handled at the environment level, not ImHex. Maybe make a script to launch ImHex, get its window, and resize it

I'd say ImHex should remember its previous window position and size when closing, including the monitor index and whether the window is maximized. This is the most flexible option and it doesn't require manual configuration.

**Reply by iTrooz** [reply]
I agree with that idea, could you open an issue about it please ?

**Reply by Calinou** [reply]
> I agree with that idea, could you open an issue about it please ?

- https://github.com/WerWolv/ImHex/issues/1215
